src/views/TimerView/TimerView.js

```diff
@@ -0,0 +1,163 @@
+
+
+import style from "./timerView.styles.css";
+import { elementFrom } from "../../shared/dom";
+
+const templateHtml = ({data}) => {
+    return `
+    <div class="clock">
+    <button class="clock-start" id="start-BTN">${data.buttonStart}</button>
+    <button class="clock-end" title="restart time">${data.buttonEnd}</button>
+    <div class="clock-display">
+        <div class="clock-displayWrap">
+            <div class="clock-displayMin">${data.displayMin}</div>
+            <div class="clock-displaySec">${data.displaySec}</div>
+        </div>
+        <div class="clock-displayWrap">
+            <p class="clock-paragraphMin clock-paragraph">${data.paragraphMin}</p>
+            <p class="clock-paragraphSec clock-paragraph">${data.paragraphSec}</p>
+        </div>
+    </div>
+    <div class="clock-btnWrap">
+        <button class="clock-btn clock-plusMin">${data.plusBtn}</button>
+        <button class="clock-btn clock-remMin">${data.minBtn}</button>
+        <button class="clock-btn clock-plusSec">${data.plusBtn}</button>
+        <button class="clock-btn clock-remSec" >${data.minBtn}</button>
+    </div>
+    <div class="clock-bckg"></div>
+</div>`
+
+}
+
+
+export const TimerView = ({renderOn, data}) => {
+    const element = elementFrom({html: templateHtml({data})});
+    console.log(element)
+
+    document.querySelector(renderOn).appendChild(element);
+    console.log(renderOn)
+
+    //display elements
+    const displayMin = document.querySelector('.clock-displayMin');
+    const displaySec = document.querySelector('.clock-displaySec');
+    
+    //background-change element
+    const clockBckg = document.querySelector('.clock-bckg');
+
+    //buttons queryselectors
+    const plusMin = document.querySelector('.clock-plusMin');
+    const remMin = document.querySelector('.clock-remMin');
+    const plusSec = document.querySelector('.clock-plusSec');
+    const remSec = document.querySelector('.clock-remSec');
+    const start = document.querySelector('.clock-start');
+    const end = document.querySelector('.clock-end');
+    const buttons = document.querySelector('.clock-btnWrap');
+
+    //adding function makeNumber for buttons
+    plusMin.addEventListener('click', makeNumber);
+    plusSec.addEventListener('click', makeNumber);
+    remMin.addEventListener('click', makeNumber);
+    remSec.addEventListener('click', makeNumber);
+    start.addEventListener('click', makeNumber);
+    end.addEventListener('click', makeNumber);
+
```

Comment:
może lepiej byłoby stworzyć osobne metody dla każdego przycisku? wtedy nie musiałbyś tworzyć wielkiej funkcji, która wewnątrz zagnieżdża ,mnóstwo ifów i odpala odpowiednie zachowanie na podstawie klasy. Dodatkowo wybraź sobie sytuację, w której ktoś wpada na pomysł zmiany nazwy klasy, bo mu się takowa nie podoba, to nie powinno wpływać na zachowanie całej aplikacji, zwłąszcza logiki.